support-workers/src/typescript/model/address.ts

```diff
@@ -0,0 +1,108 @@
+import { z } from 'zod';
+
+export const addressSchema = z.object({
+	lineOne: z.string().nullable(),
+	lineTwo: z.string().nullable(),
+	city: z.string().nullable(),
+	state: z.string().nullable(),
+	postCode: z.string().nullable(),
+	country: z.string(), //TODO: build a schema for this
+});
+
+type AddressLine = {
+	streetNumber?: string;
+	streetName: string;
+};
+
+export function combinedAddressLine(
+	addressLine1: string | null,
+	addressLine2: string | null,
+): AddressLine | undefined {
```

Comment:
is the null/undefined inconsistency here just because we're getting the incoming zod data into usable format in our code?